bin/salita

```diff
@@ -27,6 +27,8 @@ var options = require('yargs')
   .boolean('ignore-pegged').describe('ignore-pegged', 'ignore updates to packages that are pegged to a single version, rather than a range')
   .boolean('check').describe('check', 'implies --dry-run and --no-update, and returns with an exit code matching the number of updated dependencies')
   .default('check', FALSE).alias('check', 'c')
+  .string('sections').describe('sections', 'comma-separated list of sections to process, default: dep,dev,pee,opt')
```

Comment:
i believe this can be made an array instead of a string, and yargs should do the splitting for us?